src/components/PieChart/PieChart.tsx

```diff
@@ -20,81 +24,48 @@ const DONUT_WIDTH = 15;
 const HOVER_SCALE = 1.1; // duplicated in .less file
 const INNER_RADIUS = 0.5;
 
-const PieChart = createClass({
-	displayName: 'PieChart',
-
-	statics: {
-		peek: {
-			description: `
-				Pie charts are used for categorical data when you want to show the
-				relative size of each category to the whole. We use similar "x" and "y"
-				terms to keep parity with the other charts even though pie charts are
-				really just key value based.
-			`,
-			categories: ['visualizations', 'charts'],
-			madeFrom: ['ToolTip'],
-		},
-		HOVER_SCALE,
-		DONUT_WIDTH,
-		MARGIN: {
-			top: 10,
-			right: 10,
-			bottom: 10,
-			left: 10,
-		},
-	},
-
-	reducers,
-
-	propTypes: {
-		style: object`
-			Styles that are passed through to the root container.
-		`,
-
-		className: string`
-			Appended to the component-specific class names set on the root element.
-		`,
-
-		height: number`
-			Height of the chart.
-		`,
-
-		width: number`
-			Width of the chart.
-		`,
-
-		margin: shape({
-			top: number,
-			right: number,
-			bottom: number,
-			left: number,
-		})`
-			An object defining the margins of the chart. These margins typically
-			contain the axis and labels.
-		`,
-
-		data: arrayOf(object)`
-			Data for the chart. E.g.
-
-				[
-					{ x: 'Monday'    , y: 1 } ,
-					{ x: 'Tuesday'   , y: 2 } ,
-					{ x: 'Wednesday' , y: 3 } ,
-					{ x: 'Thursday'  , y: 2 } ,
-					{ x: 'Friday'    , y: 5 } ,
-				]
-		`,
-
-		hasToolTips: bool`
-			Show tool tips on hover.
-		`,
-
-		hasStroke: bool`
-			Determines if the pie slices have a stroke around them.
-		`,
-
-		palette: arrayOf(string)`
-			Takes one of the palettes exported from \`lucid.chartConstants\`.
+interface IPieChartMargin {
```

Comment:
We may end up putting this into one of the more centralized files cause this interface is used in a lot of the chart components. It's fine to leave it here for now though.